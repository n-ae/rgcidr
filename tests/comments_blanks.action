-f tests/patterns_with_comments.given tests/ips.given
