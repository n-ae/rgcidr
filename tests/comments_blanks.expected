192.168.1.1
10.0.0.1
192.168.1.100
10.0.0.255
192.168.2.50 - with comment
Log entry: 10.1.1.1 connected
2001:db8::1
2001:db8:85a3::8a2e:370:7334
